src/uniprotkb/components/entry/FunctionSection.tsx

```diff
@@ -139,6 +139,13 @@ export const CofactorView = ({ cofactors, title }: CofactorViewProps) => {
       {cofactors.map((cofactorComment, index) => (
         // eslint-disable-next-line react/no-array-index-key
         <section className="text-block" key={index}>
+          {cofactorComment.molecule && (
+            <h4 className="tiny">
+              <a href={`#${cofactorComment.molecule.replace(' ', '_')}`}>
+                {cofactorComment.molecule}:
```

Comment:
Do you think the `:` is needed? When rendered it already looks like a title to me so it's clear that follows pertains to the molecule. I know the current site does it but I'm wondering if it's necessary?

Reply:
Not sure, I matched the current website on this, but could remove if you think the new style is good enough and makes it redundant. Don't have any opinion on this

Reply:
I vote to remove the `:` as my brain is expecting something appear horizontally beside it eg

```
Isoform Mitochondrial: something about isoform mitochondrial
```